messaging/WFLY-12243_monitor_artemis_resource_adapter_thread_pools.adoc

```diff
@@ -0,0 +1,115 @@
+= Monitor Artemis resource adapter thread pools
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12243[WFLY-12243]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[X] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+== Requirements
```

Comment:
`core-size` and `max-size` metrics are missing, aren't they? They are in the description of https://issues.jboss.org/browse/EAP7-1089.

Reply:
No they are not metrics ;)

Reply:
Agree. And keep-alive-time?